### PR TITLE
Feature/typescript

### PR DESCRIPTION
**Pull request by danakt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,85 +1,78 @@
...
+$ yarn add @vkontakte/vk-mini-apps-api
+```
 
-Import necessary methods from this library instead of using [vkui-connect](https://www.npmjs.com/package/@vkontakte/vkui-connect):
+or npm
 
-``` javascript
-import VKConnect from '@vkontakte/vkui-connect-mock';
 ```
+$ npm install @vkontakte/vk-mini-apps-api
```
Comment (kuhel) [inline README.md]: @vkontakte/vk-connect-mock

File: README.md [hunk 2]
```diff
@@ -1,85 +1,78 @@
...
 
-or
+## Usage
 
-`yarn add @vkontakte/vkui-connect-mock`
+Install the library via yarn
 
-# How to use
+```
+$ yarn add @vkontakte/vk-mini-apps-api
```
Comment (kuhel) [inline README.md]: @vkontakte/vk-connect-mock

File: README.md [hunk 3]
```diff
@@ -1,85 +1,78 @@
...
-    },
-    country: {
-      id: 1,
-      title: 'UK'
-    },
-    timezone: 0
-  },
-}
+Please note that some methods may only receive (for example, `VKWebAppUpdateConfig`,
+`VKWebAppViewHide`, `VKWebAppViewRestore`, etc.). To obtain data from them you need to use the following function:
```
Comment (kuhel) [inline README.md]: To obtain data from them you need to use the event-based way and callReceiveOnlyMethod()

File: package.json [hunk 4]
```diff
@@ -1,13 +1,17 @@
 {
-  "name": "@vkontakte/vkui-connect-mock",
+  "name": "@vkontakte/vk-connect-mock",
   "version": "1.0.61",
```
Comment (kuhel) [inline package.json]: 2.0.0

File: src/mock.ts [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+    result: true
+  }),
+  VKWebAppAddToFavorites: () => ({
+    result: true
+  }),
+  VKWebAppSendPayload: () => ({
+    result: true
+  }),
+  VKWebAppGetCommunityToken: () => ({
+    access_token: 'cc9521551d93ddb290b32648a37a006d87438a67f953dd37e564eb6db1ec28f79d05c16e207f00a623ef0'
```
Comment (kuhel) [inline src/mock.ts]: yours000access111token222yours000access111token222yours000access111token2220123456789

File: src/mock.ts [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+  }),
+  VKWebAppGetCommunityAuthToken: () => ({
+    access_token: 'cc9521551d93ddb290b32648a37a006d87438a67f953dd37e564eb6db1ec28f79d05c16e207f00a623ef0'
```
Comment (kuhel) [inline src/mock.ts]: yours000access111token222yours000access111token222yours000access111token2220123456789

File: src/mock.ts [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+  }),
+  VKWebAppCommunityAccessToken: () => ({
+    access_token: 'cc9521551d93ddb290b32648a37a006d87438a67f953dd37e564eb6db1ec28f79d05c16e207f00a623ef0'
+  }),
+  VKWebAppCommunityToken: () => ({
```
Comment (kuhel) [inline src/mock.ts]: yours000access111token222yours000access111token222yours000access111token2220123456789

File: src/mock.ts [hunk 8]
```diff
@@ -0,0 +1,264 @@
...
+    result: true
+  }),
+  VKWebAppGetCommunityToken: () => ({
+    access_token: 'cc9521551d93ddb290b32648a37a006d87438a67f953dd37e564eb6db1ec28f79d05c16e207f00a623ef0'
+  }),
+  VKWebAppGetCommunityAuthToken: () => ({
+    access_token: 'cc9521551d93ddb290b32648a37a006d87438a67f953dd37e564eb6db1ec28f79d05c16e207f00a623ef0'
+  }),
+  VKWebAppCommunityAccessToken: () => ({
+    access_token: 'cc9521551d93ddb290b32648a37a006d87438a67f953dd37e564eb6db1ec28f79d05c16e207f00a623ef0'
```
Comment (kuhel) [inline src/mock.ts]: yours000access111token222yours000access111token222yours000access111token2220123456789

File: README.md (hunk 1 above)
Comment (danakt) [reply README.md]: 🤦‍♂️

**Pull request merged**